### ISSUE TITLE
Issues With War

### ISSUE THREAD

**Issue by WillisAwesome** [TownyAdvanced/Towny#3102]
This ticket template is for bug/error reports. If you are suggesting a feature and not reporting a bug please delete all the text here and begin your ticket Title with Suggestion:

### What steps will reproduce the problem?
1. Ending War
2. During War
3.

### What is the expected output?
1. When war is ended no one gets the money from the war chest even tho it says they do.
2. During war non-war members can't PVP is there a way to disable this
3. 

### What is your Towny version?
Do not say "latest". 0.93.0.0

### What is your spigot/craftbukkit version?
Do not say "latest". Spigot API 1.13.2

### Please use Pastebin.com to link the following files:

**Skipping these files is not an option. It is the easiest way to diagnose an issue ticket properly. If you do skip this step then your ticket will most likely be deleted and you will be asked to resubmit.**
1. Your full server startup from the logs\latest.log : (No Errors Sighted Very Long)
2. Your Towny config.yml (if using MySQL please remove password,) : https://pastebin.com/qvLa6Z8D
3. Your townyperms.yml : https://pastebin.com/CuygMPvQ
4. Your log's error : (No Errors in Sight)

(optional) If this is to do with permissions, 
5. Your permissions file : x

(optional) If this is to do with chat,
6. Your chatconfig.yml : x
7. Your channels.yml : x

**Comment by WillisAwesome** [comment]
I am using Vault and Essentials Economy but here is the pastebin https://pastebin.com/MrtkDN4H

**Comment by WillisAwesome** [comment]
Also about the second thing is that changeable

**Comment by LlmDl** [comment]
The second thing with pvp should be fixed by changing towns_are_neutral to false in the config's war section. 

The spoils thing, could be because no one scored any points. Your instructions on how to cause the issue isn't very clear. If you just start the war then stop it, I don't think it will pay out anything because there's no top scorer.

**Comment by WillisAwesome** [comment]
Yes but I want PVP to be enabled not just for war members is that possible without making towns apart of the war because I do have towns_are_neutral to false

**Comment by LlmDl** [comment]
Fairly sure the towns without nations wont be attackable, but their members would still be able to fight in the war itself. Your original config in the ticket OP has towns_are_neutral set to true, you've changed that?

**Comment by WillisAwesome** [comment]
Ah Alright I see now thank you
